api/src/main/java/org/pmiops/workbench/firecloud/FireCloudService.java

```diff
@@ -46,11 +47,26 @@
   void addUserToBillingProject(String email, String projectName);
 
   /**
-   * Removes the specified user from the specified billing project. Only used for errored billing
-   * projects
+   * Removes the specified user from the specified billing project.
+   *
+   * <p>Only used for errored billing projects
```

Comment:
Why `<p>`?  It these are automagically parsed to print html-y documentation, does the parser close the tags for you?

Reply:
This was the linter's doing, and the linter will actually remove a closing `</p>`.  I can remove this if I don't insist on making this a separate line.